src/NetAppFiles/NetAppFiles/VolumeGroup/NewNetAppFilesVolumeGroup.cs

```diff
@@ -0,0 +1,651 @@
+            }
+            catch
+            {
+                existingPool = null;
+            }
+            if (existingPool == null)
+            {
+                throw new AzPSResourceNotFoundCloudException($"A Pool with name '{this.PoolName}' in resource group '{this.ResourceGroupName}' does not exist. Please provide PoolName for an existing pool, use New-AzNetAppFilesPool to create a new Capacity Pool if needed.");
+            }
+            else
+            {
+                poolResourceId = existingPool.Id;
+            }
+            var volumeGroup = CreateVolumeGroup(Name, ResourceGroupName, AccountName, poolResourceId, tagPairs);
+            if (ShouldProcess(Name, string.Format(PowerShell.Cmdlets.NetAppFiles.Properties.Resources.CreateResourceMessage, ResourceGroupName)))
+            {
+                var anfVolumeGroups = AzureNetAppFilesManagementClient.VolumeGroups.Create(volumeGroup, ResourceGroupName, AccountName, Name);
+                var ret = anfVolumeGroups.ConvertToPs();
+                WriteObject(ret);
+            }
+        }
+
+        private VolumeGroupDetails CreateVolumeGroup(string name, string reourceGroup, string accountName, string poolResourceId, IDictionary<string, string> tagPairs)
+        {
+            //var defaultBackupProtocolType = BackupProtocolTypes.NSFv41;
+            //var volumeBackupProtocolTypes = BackupProtocolType == null ? defaultBackupProtocolType : BackupProtocolType;
+            VolumePropertiesExportPolicy volumeExportPolicy;
+            if (ExportPolicy == null)
+            {
+                volumeExportPolicy = new VolumePropertiesExportPolicy
+                {
+                    Rules = new List<ExportPolicyRule>()
+                    {
+                        new ExportPolicyRule { Nfsv3 = false, Nfsv41 = true, RuleIndex = 1, AllowedClients = "0.0.0.0/0", UnixReadOnly = false, UnixReadWrite = true, 
+                            Kerberos5ReadOnly = false, Kerberos5iReadOnly = false, Kerberos5iReadWrite = false, Kerberos5pReadOnly = false, 
+                            Kerberos5pReadWrite = false, Kerberos5ReadWrite = false
+                        }
+                    }
+                };
+            }
+            else
+            {
+                volumeExportPolicy = ModelExtensions.ConvertExportPolicyFromPs(ExportPolicy);
+            }
+
+            return CreateHostVolumeGroup(name, ApplicationIdentifier, poolResourceId, tagPairs, BackupProtocolType, volumeExportPolicy, StartingHostId, HostCount);
+            
+        }
+
+        private VolumeGroupDetails CreateHostVolumeGroup(string name, string sid, string poolResourceId, IDictionary<string, string> tagPairs, string[] volumeBackupProtocolTypes, VolumePropertiesExportPolicy volumeExportPolicy, int startingHostId,  int hostCount)
+        {
+            var globalPlacementRules = new List<PlacementKeyValuePairs> { new PlacementKeyValuePairs { Key = "Key1", Value = "value1" } };
```

Comment:
Why here is a hard-code GlobalPlacementRule. I saw this parameter is hiden from cmdlet, is it expected?